Sources/MapboxNavigation/CarPlaySearchController+CPSearchTemplateDelegate.swift

```diff
@@ -159,14 +75,7 @@ extension CarPlaySearchController: CPListTemplateDelegate {
 }
 
 extension GeocodedPlacemark {
-    @available(iOS 12.0, *)
-    func listItem() -> CPListItem {
-        let item = CPListItem(text: formattedName, detailText: subtitle, image: nil, showsDisclosureIndicator: true)
-        item.userInfo = [CarPlaySearchController.CarPlayGeocodedPlacemarkKey: self]
-        return item
-    }
-    
-    var subtitle: String? {
+    open var subtitle: String? {
```

Comment:
This property is necessary for displaying the address since `Geocoder.shared.geocode()` returns the address as separate from the placemarks. Can I move this to MapboxGeocoder.swift?

Reply:
MapboxGeocoder.swift would be a reasonable place to implement this functionality, since the specific logic is peculiar to the Mapbox Geocoding API. However, as a first step, we can move it to the Example application. The logic is nontrivial and not easily guessable, so we may need to include it in examples until it finds its way into MapboxGeocoder.swift. Presumably the Mapbox Search API has something similar that an application can hook into.